source/clear-linux/guides/maintenance/assign-static-ip.rst

```diff
@@ -0,0 +1,70 @@
+.. _assign-static-ip:
+
+Assign a static IP address to a network interface
+#################################################
+
+Introduction
```

Comment:
@bktan8  Please provide a couple sentences of introduction here. It can be bullets if you have a list of prerequisites.

Reply:
Please add "By default, your Clear Linux system will automatically get an IP address from your network via DHCP.  To assign your system a static IP address, follow these steps:"